src/sentry/issues/related/trace_connected.py

```diff
@@ -28,7 +28,7 @@ def trace_connected_analysis(
     if event_id:
         # If we are passing an specific event_id, we need to get the project_id
         assert project_id is not None
-        event = eventstore.backend.get_event_by_id(project_id, event_id, group_id=group.id)
+        event = eventstore.backend.get_event_by_id(int(project_id), event_id, group_id=group.id)
```

Comment:
@armenzg is going to handle the api endpoint doing the validation itself instead of here so this `int(...)` call should go away eventually (and the `extra_args` would probably become a `TypedDict` or normal parameters)